README.md

```diff
@@ -27,3 +27,9 @@ OpenSeadragon is released under the New BSD license. For details, see the [LICEN
 [github-releases]: https://github.com/openseadragon/openseadragon/releases
 [github-contributing]: https://github.com/openseadragon/openseadragon/blob/master/CONTRIBUTING.md
 [github-license]: https://github.com/openseadragon/openseadragon/blob/master/LICENSE.txt
+
+## Sponsors
+
+We are grateful for the (development or financial) contribution to the OpenSeadragon project.
+
+<img alt="Logo BBMRI ERIC" src="test/data/assets/bbmri-logo.png" height="80"/>
```

Comment:
Seems like this should be a link.

Reply:
it's relative path to the file, so that it works locally and on github as well... if I understand the question

Reply:
I am not sure about `readme-images`, IMHO cleanest solution would be top level `assets` with `assets/icons` (image viewer icons, current `image` folder) and `assets/logos`... etc, also for the future. But this changes the icon location and thus possibly introduces problems.

Reply:
Well maybe for this PR just make `assets/logos` and we'll worry about `assets/icons` later. 

By making it a link, I meant make it possible for someone viewing the README on GitHub to click on it and go to https://www.bbmri-eric.eu/.